cypress/support/dictionary/permissions.js

```diff
@@ -93,6 +93,7 @@ export default {
   checkinAll: { internal: '', gui: 'Check in: All permissions' },
   // Organization
   viewOrganization:{ internal:'', gui:'Organizations: View' },
+  OrganizationviewCreateEdit:{ internal:'', gui:'Organizations: View, edit, create' },
```

Comment:
Please start the name from the LowerCase letter

Reply:
Done